### PR TITLE
Set up a "Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by svitalis123**
# Set up a "Hello world" project as one app with Webpack
## I accomplished 

- [ ] - Creating a new Rails app called 'hello-rails-react'.
- [ ] - Using the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- [ ] - Verifing that the jsbundling-rails gem is added to your Gemfile.
- [ ] - Initializing your project with Git.
- [ ] - Installing needed node dependencies as described in the previous lesson's tutorial.
- [ ] - Creating a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- [ ] - Updating webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- [ ] - Creating a table for storing your messages and create 5 different greetings.
- [ ] - Creating  an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- [ ] - Creating a static view that will be the root of your app.
- [ ] - Creating the App component with react-router and render it in your static view.
- [ ] - Creating the Greeting component that will display a greeting. Set it up as a route in your App component.
- [ ] - Creating a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- [ ] - Displaying the random greeting in your Greeting component.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```
Comment (franclobo) [inline db/seeds.rb:7]: - [ ] Kindly add some greetings examples to run randomly when the app initializes. When I opened it I just saw a page in white.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @svitalis123, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors ✔️ 
- Good use of webpack ✔️ 
- jsbundling-rails gem is used 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/application.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from "react-redux";
+import store from './redux/configureStore';
+import App from "./App";
+
+// function App() {
+//   return (<h1>Hello World!</h1>);
+// }
```
Comment (TedLivist) [inline app/javascript/application.js:12]: - [ ] **[OPTIONAL]** I would recommend you remove this commented code.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @svitalis123,

✔️ Apps are well integrated.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**